npu.user.js

```diff
@@ -506,8 +507,11 @@ var npu = {
 					var select = $('<ul class="termSelect"></ul>');
 					var stored = npu.getUserData(null, null, ["termSelect", npu.getPage()]);
 					var found = false;
+					var admissionSemester = $("#lblTrainingName").text().split("[")[0].split(":")[1];

```

Comment:
Having a `[` in the training's name sounds so bad, but yes, we who only live by chipping away on the clientside output may never now. I shall fix it soon.

Otherwise, you find the general idea useful?